JumpscaleCore/data/bcdb/tests/11_redis_index_lost.py

```diff
@@ -20,7 +20,7 @@ def test_redis_index_lost():
 
     """
 
-    bcdb = j.data.bcdb.get("test", reset=True)
+    bcdb = j.data.bcdb.get("test_redis_index", reset=True)
```

Comment:
please don't make distinction, its not needed
bcdb needs to be cleaned using the storage layer underneath like it was